contrib/storage-indexr/pom.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0"?>
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor
+  license agreements. See the NOTICE file distributed with this work for additional
+  information regarding copyright ownership. The ASF licenses this file to
+  You under the Apache License, Version 2.0 (the "License"); you may not use
+  this file except in compliance with the License. You may obtain a copy of
+  the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required
+  by applicable law or agreed to in writing, software distributed under the
+  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+  OF ANY KIND, either express or implied. See the License for the specific
+  language governing permissions and limitations under the License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>drill-contrib-parent</artifactId>
+    <groupId>org.apache.drill.contrib</groupId>
+    <version>1.10.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>drill-indexr-storage</artifactId>
+  <name>contrib/indexr-storage-plugin</name>
+
+  <properties>
+    <indexr.version>0.1.0</indexr.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.drill.exec</groupId>
+      <artifactId>drill-java-exec</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>io.indexr</groupId>
```

Comment:
Which repositories have the dependency of index-common, indexr-segment?  I tried to build drill after applying the patch, and hit mvn dependency error.

```
[ERROR] Failed to execute goal on project drill-indexr-storage: Could not resolve dependencies for project org.apache.drill.contrib:drill-indexr-storage:jar:1.10.0-SNAPSHOT: The following artifacts could not be resolved: io.indexr:indexr-common:jar:0.1.0, io.indexr:indexr-segment:jar:0.1.0, io.indexr:indexr-server:jar:0.1.0: Could not find artifact io.indexr:indexr-common:jar:0.1.0 in cdh.repo (https://repository.cloudera.com/artifactory/cloudera-repos) -> 
```